python/lsst/summit/utils/utils.py

```diff
@@ -373,7 +372,7 @@ def fluxesFromFootprints(
     )
     if isinstance(footprints, FootprintSet):
         footprints = footprints.getFootprints()
-    elif isinstance(footprints, Iterable):
+    elif isinstance(footprints, Sequence):
```

Comment:
Why change `Iterable` to `Sequence` here?

Reply:
Mypy complains about an Iterable not being indexable in next line:
    elif isinstance(footprints, Iterable):
        if not isinstance(footprints[0], Footprint): -> Should be a sequence

Reply:
Maybe changing the code changes code behaviour...

Reply:
Ooooh, OK, then it probably _is_ a sequence rather than an iterable - shows what I know! Let's leave it as you have it then, thanks!